docs/updates/updates-10-24-2017.md

```diff
@@ -0,0 +1,170 @@
+## October 24th
+
+...
+
+[jselmani], [joel-bentley], [pradeepgangwar], [leibale], [jainsneha23], [bomsy], [nyrosmith], [stefanjudis], [rameramwe], [amelzer], [jasonLaster], [lukaszsobek], [tohmjudson], [michaelgv], [ahuth], [darkwing], [greenkeeper[bot]], [juliandescottes], [yurydelendik], [codehag], [wldcordeiro], [James300]
+
```

Comment:
we don't need greenkeeper in the here ... it might be breaking the formatting as well

Reply:
Bot rights! 😃